src/main/MessageTreeView.razor

```diff
@@ -0,0 +1,371 @@
+                                    {
+                                        if (child.Neuron.Terminal.Strength == "1")
+                                        {
+                                            @if (child.Neuron.Type == Library.Common.RelativeType.Postsynaptic)
+                                            {
+                                                <span>&#x21FE;</span>
+                                            }
+                                            else
+                                            {
+                                                <span>&#x21FD;</span>
+                                            }
+                                        }
+                                        else
+                                        {
+                                            @if (child.Neuron.Type == Library.Common.RelativeType.Postsynaptic)
+                                            {
+                                                <span>&#x290F;</span>
+                                            }
+                                            else
+                                            {
+                                                <span>&#x290E;</span>
+                                            }
+                                        }
+                                    }
+                                </div>
+                                <div onmouseover="hover('g', this.id, true, highlightNode)" onmouseout="hover('g', this.id, false, highlightNode)" class="col-auto p-1" 
+                                    id="@child.Neuron.Id" style="@(this.GetStyleByCreationAuthor(child.Neuron.Validation.IsCurrentUserCreationAuthor) + 
+                                    ((int) tagUrlType > (int) UrlType.Unrecognized ? "; max-width: calc(100% - 90px)" : string.Empty))"
+                                        userCreated="@(child.Neuron.Validation.IsCurrentUserCreationAuthor.ToString().ToLower())">
+                                    @if (child.Neuron.Validation.RestrictionReasons.Count() > 0)
+                                    {
+                                        <div class="tooltip-wrapper">
+                                            <span class="tooltip-wrapper-text">@((MarkupString) string.Join(Environment.NewLine, child.Neuron.Validation.RestrictionReasons.ToArray()))</span>
+                                            <Blazorise.Icon Name="FontAwesomeIcons.Lock" IconStyle="IconStyle.Solid" />
+                                        </div>
+                                    }
+                                    @if (!string.IsNullOrWhiteSpace(child.Neuron.Tag))
+                                    {
+                                        @if (child.Seen)
+                                        {
+                                            child.Neuron.Tag = ConvertDriveLinkToDirectLink(child.Neuron.Tag);
```

Comment:
Do we need to assign the result to ConvertDriveLinkToDirectLink? Can we create a variable, assign to it, and use it in the "a" and "img" elements in the following two lines?